packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts

```diff
@@ -1115,6 +1149,10 @@ const getEchartAxes = ({
                         : showGridX,
                 },
                 inverse: !!xAxisConfiguration?.[0].inverse,
+                data: useWeekAxisConfiguration ? customXRange : undefined,
```

Comment:
We need to apply this to all necessary axis.
example when I flip axis

<img width="1000" alt="Screenshot 2024-05-03 at 10 05 27" src="https://github.com/lightdash/lightdash/assets/9117144/4c9c1ac2-c3b7-47e2-bdfd-86dcbd213fa7">

Reply:
Good call. I meant to do that.

Reply:
Done